src/components/hex-map/index.js

```diff
@@ -61,8 +62,16 @@ const mapDispatchToProps = (dispatch, props) => ({
   addRouteLocation: key => dispatch(addRouteLocation(key)),
   completeRoute: () => dispatch(completeRoute(props.intl)),
   updateNavSettings: (key, value) => dispatch(updateNavSettings(key, value)),
+  toEntity: (entityType, entityId) => {
+    const route = `/sector/${
+      props.match.params.sector
+    }/${entityType}/${entityId}`;
```

Comment:
Not a fan of all this logic in the react-router definition for this component. Will probably move this some day.